### PR TITLE
Client API Configuration for an Application

### PR DESCRIPTION
**Pull request by cilim**
Solves https://github.com/stormpath/stormpath-sdk-ruby/issues/192

### CODE REVIEW

File: spec/resource/application_spec.rb [hunk 1]
```diff
@@ -108,10 +137,21 @@
...
   end
 
+  describe 'edit authorized_origin_uris' do
+    let(:authorized_origin_uris) { ['https://dnsLabel1.apps.stormpath.io', 'https://dnsLabel2.apps.stormpath.io'] }
+
+    it 'changes authorized origin uris on application' do
+      application.authorized_origin_uris = authorized_origin_uris
+      response = application.save
+
+      expect(response).to eq application
```
Comment (DamirSvrtan) [inline spec/resource/application_spec.rb]: check this if it's needed.

File: spec/resource/application_web_config_spec.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        expect(web_config).to respond_to("#{endpoint}=")
+        expect(web_config.send(endpoint)).to be_a Hash
+      end
+    end
+  end
+
+  describe 'enabling/disabling endpoints' do
+    it 'should be able to switch to enabled and disabled' do
+      Stormpath::Resource::ApplicationWebConfig::ENDPOINTS.each do |endpoint|
+        instance_eval("web_config.#{endpoint}= { enabled: false }")
```
Comment (DamirSvrtan) [inline spec/resource/application_web_config_spec.rb]: web_config.send("#{endpoint_name}=", { enabled: false })

File: spec/resource/application_web_config_spec.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        web_config.save
+
+        expect(instance_eval("application.web_config.#{endpoint}['enabled']")).to eq false
```
Comment (DamirSvrtan) [inline spec/resource/application_web_config_spec.rb]: `application.web_config.send(endpoint).send([], 'enabled')`

**Comment by cilim** [comment]
Comments fixed @DamirSvrtan

**Comment by DamirSvrtan** [comment]
If this passes, feel free to merge it!

**Pull request merged**